### DISCUSSION TITLE
Lora Problem

### DISCUSSION THREAD

**Discussion by Abdoul2424** [acheong08/Diffusion-ColabUI#119] [Q&A] [answered]
The problem with the colab is that it downloaded the LoRA first which is a problem because then it gets stuck in the stable diffusion folder instead of going to the LoRA folder, is there a way I can make it where it does that after setting up WebUi since the folders like the LoRA folder are created after WebUi is launched, thx

**Comment by acheong08** [comment]
There is a checkbox to configure where the LoRAs go

**Reply by acheong08** [reply, marked as answer]
You can also drag and drop the LoRA from their folder to the actual location it's meant to be at

**Reply by Abdoul2424** [reply]
> You can also drag and drop the LoRA from their folder to the actual location it's meant to be at

I already know about that way but I want to do it in a more efficient way

>There is a checkbox to configure where the LoRAs go

And where might that checkbox be exactly? And what is it called? 
Here are the options that I can see that are related to LoRA:
save_to_gdrive
use_gdrive_loras
gdrive_loras_path
I don't want to use the Google Drive Method since I am using the model link in order to install the model

**Reply by acheong08** [reply]
It's inside the UI for CivitAI

**Reply by Abdoul2424** [reply]
> It's inside the UI for CivitAI

Oh ok thank you so much for the quick reply I appreciate it
